From 09bc0003ad0da11030aad93532184bab2fad6bb9 Mon Sep 17 00:00:00 2001
From: Razvan Stefanescu - M50855 <Razvan.Stefanescu@microchip.com>
Date: Tue, 18 Jun 2019 16:07:38 +0300
Subject: [PATCH] Fwd: [PATCH] fix warnings

Received: from MWHPR1101MB2239.namprd11.prod.outlook.com (2603:10b6:800::45)
 by SN6PR11MB3069.namprd11.prod.outlook.com with HTTPS via
 SN2PR01CA0077.PROD.EXCHANGELABS.COM; Tue, 18 Jun 2019 13:06:59 +0000
Received: from BN6PR11CA0061.namprd11.prod.outlook.com (2603:10b6:404:f7::23)
 by MWHPR1101MB2239.namprd11.prod.outlook.com (2603:10b6:301:5b::8) with
 Microsoft SMTP Server (version=TLS1_2,
 cipher=TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384) id 15.20.1987.10; Tue, 18 Jun
 2019 13:06:57 +0000
Received: from BY2NAM01FT046.eop-nam01.prod.protection.outlook.com
 (2a01:111:f400:7e42::202) by BN6PR11CA0061.outlook.office365.com
 (2603:10b6:404:f7::23) with Microsoft SMTP Server (version=TLS1_2,
 cipher=TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384) id 15.20.1987.12 via Frontend
 Transport; Tue, 18 Jun 2019 13:06:57 +0000
Authentication-Results: spf=temperror (sender IP is 198.175.253.82)
 smtp.mailfrom=microchip.com; microchiptechnology.mail.onmicrosoft.com;
 dkim=none (message not signed)
 header.d=none;microchiptechnology.mail.onmicrosoft.com; dmarc=temperror
 action=none header.from=microchip.com;
Received-SPF: TempError (protection.outlook.com: error in processing during
 lookup of microchip.com: DNS Timeout)
Received: from email.microchip.com (198.175.253.82) by
 BY2NAM01FT046.mail.protection.outlook.com (10.152.68.224) with Microsoft SMTP
 Server (version=TLS1_2, cipher=TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256) id
 15.20.1987.11 via Frontend Transport; Tue, 18 Jun 2019 13:06:55 +0000
Received: from chn-vm-ex01.mchp-main.com (10.10.87.71) by
 chn-vm-ex03.mchp-main.com (10.10.87.152) with Microsoft SMTP Server
 (version=TLS1_2, cipher=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256) id
 15.1.1713.5; Tue, 18 Jun 2019 06:06:41 -0700
Received: from rob-ult-m50855.microchip.com (10.10.85.251) by
 chn-vm-ex01.mchp-main.com (10.10.85.143) with Microsoft SMTP Server id
 15.1.1713.5 via Frontend Transport; Tue, 18 Jun 2019 06:06:40 -0700
From: Razvan Stefanescu <razvan.stefanescu@microchip.com>
To: Eugen Hristev - M18282 <Eugen.Hristev@microchip.com>
CC: Razvan Stefanescu <razvan.stefanescu@microchip.com>
Subject: [PATCH] fix warnings
Date: Tue, 18 Jun 2019 16:06:56 +0300
Message-ID: <20190618130657.23300-1-razvan.stefanescu@microchip.com>
X-Mailer: git-send-email 2.20.1
Content-Transfer-Encoding: 8bit
Return-Path: Razvan.Stefanescu@microchip.com
Content-Type: text/plain
X-CrossPremisesHeadersFilteredBySendConnector: chn-vm-ex03.mchp-main.com
X-OrganizationHeadersPreserved: chn-vm-ex03.mchp-main.com
X-MS-Exchange-Organization-ExpirationStartTime: 18 Jun 2019 13:06:55.4669
 (UTC)
X-MS-Exchange-Organization-ExpirationStartTimeReason: OriginalSubmit
X-MS-Exchange-Organization-ExpirationInterval: 1:00:00:00.0000000
X-MS-Exchange-Organization-ExpirationIntervalReason: OriginalSubmit
X-MS-Exchange-Organization-Network-Message-Id:
 ffae1b09-4eb6-4cf4-bfff-08d6f3edd6b7
X-EOPAttributedMessage: 0
X-MS-Exchange-Organization-MessageDirectionality: Originating
X-Forefront-Antispam-Report:
 CIP:198.175.253.82;IPV:NLI;CTRY:US;EFV:NLI;SFV:NSPM;SFS:(2980300002)(189003)(199004)(7696005)(51416003)(6666004)(356004)(6916009)(63350400001)(246002)(50466002)(36756003)(53416004)(7636002)(486006)(44832011)(63370400001)(450100002)(305945005)(4326008)(126002)(476003)(22756006)(48376002)(2616005)(6636002)(72206003)(336012)(5660300002)(37006003)(14444005)(86362001)(8676002)(50226002)(47776003)(1096003)(77096007)(8936002)(1076003)(186003)(26005)(2870700001);DIR:INB;SFP:;SCL:1;SRVR:MWHPR1101MB2239;H:email.microchip.com;FPR:;SPF:TempError;LANG:en;PTR:smtpout.microchip.com;MX:1;A:1;
X-MS-Exchange-Organization-AuthSource:
 BY2NAM01FT046.eop-nam01.prod.protection.outlook.com
X-MS-Exchange-Organization-AuthAs: Anonymous
X-OriginatorOrg: microchip.com
X-MS-PublicTrafficType: Email
X-MS-Office365-Filtering-Correlation-Id: ffae1b09-4eb6-4cf4-bfff-08d6f3edd6b7
X-Microsoft-Antispam:
 BCL:0;PCL:0;RULEID:(2390118)(7020095)(4652040)(8989299)(4534185)(4627221)(201703031133081)(201702281549075)(8990200)(5600148)(711020)(4605104)(1401327)(2017052603328);SRVR:MWHPR1101MB2239;
X-MS-TrafficTypeDiagnostic: MWHPR1101MB2239:
X-MS-Oob-TLC-OOBClassifiers: OLM:125;
X-MS-Exchange-Organization-SCL: 1
X-MS-Exchange-CrossTenant-OriginalArrivalTime: 18 Jun 2019 13:06:55.4138
 (UTC)
X-MS-Exchange-CrossTenant-Network-Message-Id: ffae1b09-4eb6-4cf4-bfff-08d6f3edd6b7
X-MS-Exchange-CrossTenant-Id: 3f4057f3-b418-4d4e-ba84-d55b4e897d88
X-MS-Exchange-CrossTenant-OriginalAttributedTenantConnectingIp: TenantId=3f4057f3-b418-4d4e-ba84-d55b4e897d88;Ip=[198.175.253.82];Helo=[email.microchip.com]
X-MS-Exchange-CrossTenant-FromEntityHeader: HybridOnPrem
X-MS-Exchange-Transport-CrossTenantHeadersStamped: MWHPR1101MB2239
X-MS-Exchange-Transport-EndToEndLatency: 00:00:03.6259037
X-MS-Exchange-Processed-By-BccFoldering: 15.20.1987.013
X-Microsoft-Antispam-Mailbox-Delivery:
 ucf:0;jmr:0;ex:0;auth:0;dest:I;ENG:(20160514016)(750119)(520011016)(944506303)(944626516);
X-Microsoft-Antispam-Message-Info:
 s0etTas3Jm0sqWii/uje2xkad6osb+UW2LhnuHHRnK375gzZp8iIuee5ij7AxXAct7Bbyy+rdDHWy/oFkVpzuZX8z7U91NlzLKmT1x5Cmhk48YOUMmhcMD2JPnI+f3adC5fRtVVfpjLaqqZ9Cd4cZXVRRh+XwioSerRtxJwtbRHq8SGf1unqE8lAl/bLRAjEKPX5AR3sjzYXpaaaylV2TDG4oFed750ytxrMHauL0hFFGCEMvKXh7x/a2p8D14emNTfIrfG9uoH5mWlWIhOOkD/VvESu+nEfTaoAfyJDvlle8PdieVJAmLAW6cr7J/9qfbaOVZWCVslVTpJOZvZVo2AK1blVx3U1696DLoocakaRB3eC7VIwxHAyUBzUEmlME/5YjKuyNwgZW7ywcP/F0rLKyQYsd50FYRpx6ancFUYHLRuFWWDVKXAtnflZQhYxUmQS9yX1PN5cxxfFN1XsqmwokbOOTbd0jwUR4aCGkIoCUWHGJOTLrT0Yl1dYvO4Wy6OMpuI0taOUgtahRxzma/spDmmaB15uNUG/4StF7jk4gtX3QnadCis412Zfp4aXelTUNZV/0ZQv10HABIOdMVLlRE9KZVSToHIS5OhlGSl97sk10YcDY+ECoUAzlevcccaUxqPEYM5eX7IYpOeHWvUFY8bbgA998RtOzq3T8PBPMS4o32uHkxiDzeuKWbdZf8LTcHAEosreh6fAECzQB9VQQ26aZ3Z5ZG60RM97f7AHrTgSMFUF3nfxvCRW7k5TNffGuI8D1w9dtmQ+v7AVmMIUZN1svq141yZzIhyek8KPT0+J8tDW6vm+PQVMW55ycec1OLR9MV97fEvWtY43Tpw3XOaDM42Ys/SQbcfJklyGfaWvKl9YzQ5Y1TBcdz1T4jDtg+xK2AGoBqymjSusww==
MIME-Version: 1.0

../atmel-video-player-1.9/videoplayer.cpp: In member function 'bool VideoPlayer::createPipeline()':
../atmel-video-player-1.9/videoplayer.cpp:169:11: warning: variable 'bus_watch_id' set but not used [-Wunused-but-set-variable]
     guint bus_watch_id;
           ^~~~~~~~~~~~
../atmel-video-player-1.9/videoplayer.cpp: In member function 'bool VideoPlayer::play(bool, int)':
../atmel-video-player-1.9/videoplayer.cpp:225:1: warning: no return statement in function returning non-void [-Wreturn-type]
 }
 ^
../atmel-video-player-1.9/videoplayer.cpp: In member function 'bool VideoPlayer::pause()':
../atmel-video-player-1.9/videoplayer.cpp:229:1: warning: no return statement in function returning non-void [-Wreturn-type]
 }
 ^

Signed-off-by: Razvan Stefanescu <razvan.stefanescu@microchip.com>
---
 videoplayer.cpp | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/videoplayer.cpp b/videoplayer.cpp
index 63af9e0..19f40a3 100644
--- a/videoplayer.cpp
+++ b/videoplayer.cpp
@@ -156,7 +156,6 @@ bool
 VideoPlayer::createPipeline(){
     GError *error = NULL;
     GstBus *bus;
-    guint bus_watch_id;
     /* Make sure we don't leave orphan references */
     destroyPipeline();
 
@@ -186,7 +185,7 @@ VideoPlayer::createPipeline(){
     }
 
     bus = gst_pipeline_get_bus (GST_PIPELINE (this->_videoPipeline));
-    bus_watch_id = gst_bus_add_watch (bus, &busCallback, this);
+    gst_bus_add_watch (bus, &busCallback, this);
     gst_object_unref (bus);
 
     return true;
@@ -212,10 +211,12 @@ bool VideoPlayer::play(bool mute,int volume){
     emit playState(this->setState(GST_STATE_PLAYING));
     setVolume(volume);
     setMute(mute);
+    return true;
 }
 
 bool VideoPlayer::pause(){
     emit playState(!this->setState(GST_STATE_PAUSED));
+    return true;
 }
 
 bool VideoPlayer::null(){
-- 
2.7.4

